modules/ossm-cert-manager-install-istio-resource.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-istio-resource_{context}"]
+= Install your Istio resource
```

Comment:
```suggestion
= Installing your Istio resource
```
Procedure titles typically start with gerunds. If you accept this revision, be sure to change any references to the title as well. See https://www.ibm.com/docs/en/ibm-style?topic=format-headings#wording.